account_asset_analytic/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# flake8: noqa
```

Comment:
OK, I'll apply same Flake8 configuration of Travis to my local environment.
But AFAIK, this pragma is only disabling checking in ```__init__.py``` file, not all models